### PR TITLE
Scene action toggle

### PR DESCRIPTION
**Pull request by rpochet**
### Pull Request check-list

To ensure your Pull Request can be accepted as fast as possible, make sure to review and check all of these items:

- [ x] If your changes affects code, did your write the tests?
- [ x] Are tests passing? (`npm test` on both front/server)
- [ x] Is the linter passing? (`npm run eslint` on both front/server)
- [ x] Did you run prettier? (`npm run prettier` on both front/server)
- [ x] If you are adding a new features/services, did you run integration comparator? (`npm run compare-translations` on front)
- [ x] Did you test this pull request in real life? With real devices? If this development is a big feature or a new service, we recommend that you provide a Docker image to [the community](https://community.gladysassistant.com/) for testing before merging.
- [ ] If your changes modify the API (REST or Node.js), did you modify the API documentation? (Documentation is based on comments in code)
- [ ] If you are adding a new features/services which needs explanation, did you modify the user documentation? See [the GitHub repo](https://github.com/GladysAssistant/v4-website) and the [website](https://gladysassistant.com).
- [ ] Did you add fake requests data for the demo mode (`front/src/config/demo.js`) so that the demo website is working without a backend? (if needed) See [https://demo.gladysassistant.com](https://demo.gladysassistant.com).

NOTE: these things are not required to open a PR and can be done afterwards / while the PR is open.

### Description of change

Add action toggle ON <> OFF to scene

### CODE REVIEW

**Comment by Pierre-Gilles** [comment]
@rpochet Tell me when this is ready for review :)

**Comment by rpochet** [comment]
C'est bon pour moi

File: front/src/config/i18n/fr.json [hunk 1]
```diff
@@ -1076,12 +1076,18 @@
       "turnOffLights": {
         "label": "Sélectionnez les lumières que vous souhaitez éteindre"
       },
+      "toggleLights": {
+        "label": "Sélectionnez les lumière que vous souhaitez inverser"
```
Comment (Pierre-Gilles) [inline front/src/config/i18n/fr.json]: *lumières

**Review by Pierre-Gilles — CHANGES_REQUESTED** [review]
Thanks for this PR 🙏

it works fine on my side, good job! 

Screenshot 2022-12-18 at 16 13 45

Just a little UX feedback, maybe we should explain in the action what does this action do? 

Screenshot 2022-12-18 at 16 10 16

Like a little text saying "Si les lumières sont allumées, elle seront éteintes, et vice-versa".

What do you think?

File: front/src/config/i18n/fr.json (hunk 1 above)
Comment (rpochet) [reply front/src/config/i18n/fr.json]: it is true that in french it is not so pretty...

What about this?
image

But OK for a more readable little text

Comment (Pierre-Gilles) [reply front/src/config/i18n/fr.json]: It's a bit dirty to have this long text in bold, what about having a shorter text in bold and an explanation in small ?

Comment (rpochet) [reply front/src/config/i18n/fr.json]: I keep the same layout as in other actions. Should not be part of a new PR?

Comment (Pierre-Gilles) [reply front/src/config/i18n/fr.json]: Yeah but other actions are not that long, so it's a problem for this one! 

Can be a conditional translations if you still want to re-use the same components

**Review by Pierre-Gilles — APPROVED** [review]
Good for me now with the changes! Thanks for taking the time 🙏

**Pull request merged**